what-our-users-say.html

```diff
@@ -247,7 +247,19 @@ <h5 style="font-weight: normal">Let's continue to build our community together.<
                         "At Codewave, we help SME’s and Enterprises across the globe, modernise their existing applications or build robust new innovative digital initiatives from the ground up and Loopback has "got our back" on more than one occasions, bringing rapid innovation without compromising on security or scalability."
                         <br>
                         <br>
-                        <b> Abhijith HK, Founder & CEO, Codewave </b>
+                        <b>Abhijith HK, Founder & CEO, Codewave</b>
+                    </div>
+                    </a>
+                </div>
+            </div>
+            <div class="row text-center" style="justify-content: center">
+                <div class="col-sm-12 col-md-6">
+                    <a class="card-2">
+                    <div class="row" style="padding:10px">
+                        "At DayOne, we put customers first and have a strong commitment to deliver the best quality technology solutions with fast-pace velocity. Therefore, a reliable, secured, well-maintained, powerful and performant back-end framework is must! Loopback just simply exceeds our expectation! Having successfully built and delivered dozens of digital products and platforms for e-learning, crypto, e-commerce, food delivery, travel, real estate platforms and many digital transformation projects for both startups and enterprises using LoopBack, we know this is still gonna be our go-to framework to build further amazing products in the future! Using LoopBack for year from the early day of version 2 until current version 4, I also highly appreciate the dedication of LoopBack team for their passion, long-term commitment, and caring of the community for a very long time. Thank you so much!"
```

Comment:
```suggestion
                        "At DayOne, we put customers first and have a strong commitment to deliver the best quality technology solutions with fast-pace velocity. Therefore, a reliable, secured, well-maintained, powerful and performant back-end framework is must! LoopBack just simply exceeds our expectation! Having successfully built and delivered dozens of digital products and platforms for e-learning, crypto, e-commerce, food delivery, travel, real estate platforms and many digital transformation projects for both startups and enterprises using LoopBack, we know this is still gonna be our go-to framework to build further amazing products in the future! Using LoopBack for year from the early day of version 2 until current version 4, I also highly appreciate the dedication of LoopBack team for their passion, long-term commitment, and caring of the community for a very long time. Thank you so much!"
```

Reply:
@dhmlau I updated all `Loopback` in the new commit :)